commands/CloudCommands.js

```diff
@@ -210,7 +210,8 @@ CloudCommand.prototype = extend(BaseCommand.prototype, {
 					choices: [
 						'Photon',
 						'Core',
-						'P1'
+						'P1',
+						'Electron'
```

Comment:
We'll gonna add them anyway so one thing less to remember then. Missing binary is gonna get caught in `prompt` callback so this doesn't cause code to crash.